akka-docs/src/test/scala/docs/stream/IntegrationDocSpec.scala

```diff
@@ -134,6 +131,35 @@ object IntegrationDocSpec {
   }
   //#ask-actor
 
+  object MessageProcessor {
+    case object Ack
+    case object Init
+    case object GetTotalProcessedMessage
+    case class ProcessMessage(msg: Int)
+    case class TotalProcessedMessage(count: Long)
+    case object Completed
+  }
+  class MessageProcessor extends Actor {
+    import MessageProcessor._
+    var totalMessageProcessed = 0
+
+    override def receive: Receive = {
+      case Init ⇒
+        sender() ! Ack
+
+      case ProcessMessage(msg) ⇒
+        totalMessageProcessed += 1
+        println(s"received message: $msg")
+        sender() ! Ack
+
+      case GetTotalProcessedMessage ⇒
+        sender() ! TotalProcessedMessage(totalMessageProcessed)
+
+      case Completed ⇒
+        println(s"Stream processing completed ")
+    }
+  }
```

Comment:
Shouldn't the actor also be included in the sample? I see the docs referring to #actorRefWithAckActor but no tags here?

Reply:
oh thanks

Reply:
that was a great catch, actually 2 tests showed the same thing, but one used the recevier actor and one this one; I removed the test and this actor -- the other shows things well already